dev/tests/integration/testsuite/Magento/Sales/_files/quote_with_backorder.php

```diff
@@ -0,0 +1,73 @@
+<?php
```

Comment:
No quote_with_backorder_rollback.php file? Or, I missed it?

(Rollbacks would undo/erase all data added with this fixture)

Reply:
Arghh, I knew I missed something. Right. I'm on it.

Reply:
Committed.